### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SerhiiUnhurian**
- [DEMO LINK](https://SerhiiUnhurian.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://SerhiiUnhurian.github.io/layout_search-bar-airbnb/report/html_report/)

file:///Users/serhiiunhurian/projects/layout_search-bar-airbnb/backstop_data/html_report/index.html

please check the structure, i need advice what is wrong

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+  color: #3d4e61;
+}
+
+.page {
+  margin: 0;
+  box-sizing: border-box;
+}
```
Comment (Esceype) [inline src/style.css:20]: Add here padding 8px for inline paddings

**Review by Esceype — CHANGES_REQUESTED** [review]
Overall great job!!! Add the tests report link to the description of your PR and check the comments

File: src/index.html [hunk 2]
```diff
@@ -10,7 +10,46 @@
...
+  <body class="page">
+    <div class="search-bar" data-qa="big">
+      <form
+        action="/"
+        class="search-bar__form search-bar__form--big"
+        method="post"
+      >
+        <label>
+          <div class="icon icon--big"></div>
+        </label>
```
[the remark this answers is not in the document]
Comment (SerhiiUnhurian) [reply src/index.html:22]: okay changed to img)
but i didn't wrap it in one tag, should i do it? like in one lable or div?

File: src/style.css (hunk 1 above)
Comment (SerhiiUnhurian) [reply src/style.css:20]: got it

**Comment by SerhiiUnhurian** [comment]
i don't know how to set dependency when input is active or input is active to change background color

File: src/index.html [hunk 3]
```diff
@@ -10,7 +10,44 @@
...
+  <body class="page">
+    <div class="search-bar">
+      <form
+        action="/"
+        class="search-bar__form search-bar__form--big"
+        method="post"
+        data-qa="big"
+      >
+        <img class="icon icon--big" src="./images/Search.svg" alt="Search img">
+        <label>
```
Comment (Andriy-Shymkiv) [inline src/index.html:22]: why do you need this label without any attributes? I think you can remove it, and from small input too

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+/* .search-bar--small {
+  margin-top: 20px;
+} */
```
Comment (Andriy-Shymkiv) [inline src/style.css:36]: remove comments

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+.search-bar:active {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-bar__form {
+  display: flex;
+  align-items: center;
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:51]: why do you need form to be a flexbox?

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+  height: 11px;
+  width: 11px;
+  margin: 0 9px 0 13px;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  border: 0 solid;
+  display: flex;
+  align-items: center;
```
Comment (Andriy-Shymkiv) [inline src/style.css:83]: I don't think that you need flexbox here

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+.icon {
+  background-position: center;
+  background-size: cover;
+  color: #687c94;
+}
+
+.icon--big {
+  height: 19px;
+  width: 19px;
+  margin: 0 16px 0 26px;
```
Comment (Andriy-Shymkiv) [inline src/style.css:70]: for now it's okay, but you can use position property for cases like this instead of margins

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+.search-bar {
+  margin: 20px 0 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
```
Comment (Andriy-Shymkiv) [inline src/style.css:30]: it is white by default, no need to set it here

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+  color: #3d4e61;
+}
+
+.page {
+  margin: 0;
+  padding-inline: 8px;
```
Comment (Andriy-Shymkiv) [inline src/style.css:19]: `padding-inline` ?? maybe let's just not reset the default margins instead?

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+  line-height: normal;
+}
+
+.input:focus {
+  font-family: Avenir, sans-serif;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 900;
+  line-height: normal;
```
Comment (Andriy-Shymkiv) [inline src/style.css:104]: it is normal by default

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input--small:focus {
+  font-family: Avenir, sans-serif;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  font-size: 14px;
+  font-style: normal;
```
Comment (Andriy-Shymkiv) [inline src/style.css:112]: same here

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+  font-weight: 900;
+  line-height: normal;
```
Comment (Andriy-Shymkiv) [inline src/style.css:114]: and here

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+.input:focus {
+  font-family: Avenir, sans-serif;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 900;
+  line-height: normal;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```
Comment (Andriy-Shymkiv) [inline src/style.css:105]: you use this background twice, so lets create a variable for it. does it makes sense?

File: src/style.css [hunk 14]
```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+  color: #3d4e61;
```
Comment (Andriy-Shymkiv) [inline src/style.css:6]: move this color to some variable and reuse it

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
good job. need some improvements

**Comment by SerhiiUnhurian** [comment]
i redid the app, please check it, i forgot to make a variable for the color in the last commit

File: src/style.css [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+/* .search-bar:active {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+} */
```
Comment (denys-danyliuk) [inline src/style.css:29]: delete comments

File: src/index.html [hunk 16]
```diff
@@ -10,7 +10,38 @@
...
+    <label>
+      <form
+        class="search-bar search-bar--small"
+        action="/"
+        method="post"
+        data-qa="small"
+      >
+        <input
+          class="input input--small"
+          name="search"
```
Comment (denys-danyliuk) [inline src/index.html:40]: each element should have different value in name attribute

File: src/style.css [hunk 17]
```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir-Book, sans-serif;
```
Comment (denys-danyliuk) [inline src/style.css:2]: add same font-family, 7'th point in checklist

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 17 above)
Comment (SerhiiUnhurian) [reply src/style.css:2]: @font-face {
  font-family: Avenir, sans-serif;
  src: url(fonts/Avenir-Book.ttf);
  font-weight: 300;
  color: #3d4e61;
}

@font-face {
  font-family: Avenir, sans-serif;
  src: url(fonts/Avenir-Heavy.ttf);
  font-weight: 900;
  color: #3d4e61;
}

типу так зробити?

File: src/style.css (hunk 15 above)
Comment (SerhiiUnhurian) [reply src/style.css]: okay

File: src/index.html (hunk 16 above)
Comment (SerhiiUnhurian) [reply src/index.html]: my input is not working without label tag also

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**